hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -257,6 +269,14 @@ public String getFrameworkHostAddress() {
     return hostAddress;
   }
 
+  public String getConfigServerHost() {
+    String configServerHost = System.getProperty("mesos.hdfs.config.server.host");
```

Comment:
We should be following the same format as the other config values here instead of using System.getProperty